backend/routes.js

```diff
@@ -17,4 +17,26 @@ router.route('/users/:user_id')
     .patch(userController.update)
     .delete(userController.delete);
 
-module.exports = router;
\ No newline at end of file
+const productController = require('./controllers/product');
+
+// Order Routes
```

Comment:
Product Routes ;)

Reply:
No idea how this could have happened... :D